src/main/java/Duke.java

```diff
@@ -1,10 +1,129 @@
+import java.util.Arrays;
+import java.util.Scanner;
 public class Duke {
+
+    public static Task[] addTask(Task[] tasks, Task task){
```

Comment:
maybe you can have an alternate naming for `task` since 6/8 words are some variation of "task"